src/Microsoft.PowerApps.TestEngine/ITestEngineEvents.cs

```diff
@@ -16,6 +16,11 @@ public interface ITestEngineEvents
         /// </summary>
         public void SetNumberOfTotalCases(int numCases);
 
+        /// <summary>
+        /// Logs player type for our purposes
+        /// </summary>
+        public void LegacyPlayerType(bool value);
```

Comment:
We're going to have to think about how to handle breaking changes like this to the interface... This is a public interface, so if anyone is taking a dependency on this library outside of our team then this could cause some issues. In this case, do we actually need to log this to the console or could this just go in the telemetry? Is this valuable to the customer to know?

Reply:
Would it be better to have a "Configuration" object that could be passed which we could add config properties like this without needing to worry about breaking changes?

Reply:
One other way to do this is in  .github\workflows\yaml-integration-tests.yml 
- Right now there is one integration tests step, instead separate it to two to run integration tests for legacy and onePlayer .
- This way might be cleaner and always later easy to remove one older version is deprecated.
<img width="533" alt="image" src="https://github.com/microsoft/PowerApps-TestEngine/assets/98551644/3fe8d672-5ad3-45de-95d2-16754620c492">